packages/app-store/paypal/lib/Paypal.ts

```diff
@@ -197,12 +198,14 @@ class Paypal {
         body: JSON.stringify(body),
       });
 
-      if (response.ok) {
-        const result = await response.json();
-        return result.id as string;
+      if (!response.ok) {
+        throw new Error(await response.json());
```

Comment:
Should we add a message string in here, something like: 
```
const message = `${response.statusText}: ${JSON.stringify(await response.json())}`;
throw new Error(message);
```

Reply:
Done